src/test/java/org/fundacionjala/sfdc/stepdefinitions/AccountsSteps.java

```diff
@@ -102,6 +104,6 @@ public void iClickOnDeleteFromAccount() {
     @Then("^the Account should not be displayed on Home Page$")
     public void theAccountShouldNotBeDisplayedOnHomePage() {
         AccountHome accountHome = new AccountHome();
-        assertFalse(accountHome.isDisplayedItem(map.get(AccountFormField.ACCOUNT_NAME)));
+        helper.getAssertion().assertFalse(accountHome.isDisplayedItem(map.get(AccountFormField.ACCOUNT_NAME)));
```

Comment:
please add a variable for helper.getAssertion()

Reply:
It's okay, thanks for the observation.

Reply:
Should I add a global or local variable?
Should I do this on all asserts?

Reply:
OK I did it.